device/dell/x86_64-dellemc_s5248f_c3538-r0/DellEMC-S5248f-P-25G/sesto-1.json

```diff
@@ -0,0 +1,256 @@
+{
+  "lanes": [
+    {
+      "index": 200,
+      "local_lane_id": 0,
```

Comment:
what is local_lane_id, why are they same for different lanes?

Reply:
The lane id's are not currently used and are a placeholder as recommended by the original SAI_Gearbox_API_Proposal documentation.